.gitignore

```diff
@@ -1,2 +1,5 @@
 dist.js
+.DS_Store
```

Comment:
That's what I thought when I put there. Many many people doesn't have .DS_Store in global .gitignore